src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Features/CannotConfirmSteps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Net;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using FluentAssertions;
+using SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests.Features
+{
+    [Binding]
+    [Scope(Feature = "CannotConfirm")]
+    public class CannotConfirmSteps : StepsBase
+    {
+        private readonly TestContext _context;
+        private readonly RegisteredUserContext _userContext;
+        private readonly long _apprenticeshipId;

```

Comment:
It didn't exist when you started this so you couldn't have used it, but *now* there is a `HashedId` that lets takes care of the hashing for you.  See here: https://github.com/SkillsFundingAgency/das-apprentice-commitments-web/blob/9055dfc0d68ee04b2c3fbd76036255424d729e9b/src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Features/ConfirmYourTrainingProviderSteps.cs#L28